samples/start-app/python-fastapi/app/persistence/userRepository.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy.orm import Session
+
+from . import models
+from app.model import schemas
+
+def get_user(db: Session):
+    return db.query(models.User).order_by(models.User.id.desc()).first()
+
+def get_user_by_sub(db: Session, sub):
+    return db.query(models.User).filter(models.User.sub == sub).first()
+
+def create_user(db: Session, user: schemas.UserCreate):
+    user_from_db = get_user(db);
+    db_user = models.User(
+        email = user.email,
+        firstname = user.firstname,
+        lastname = user.lastname,
+        sub = user.sub
+    )
+    if user_from_db is not None and user_from_db.sub == user.sub:
```

Comment:
I am not certain to understand exactly what happen here 🤔
You get the first user from db and compare it with given user ?

Reply:
I don't agree. You could have an app with multiple users. It's not the same as token.

Reply:
fixed